test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -182,25 +181,28 @@ describe('Deployments View widget', () => {
     });
 
     describe('with filters', () => {
+        const deploymentNameThatMatchesFilter = `${specPrefix}precious_deployment`;
         const filterId = 'only-precious';
+        const filterRules = [
```

Comment:
Optional:

Could we add a type annotation?

```suggestion
        const filterRules: Stage.Common.Filters.Rule[] = [
```